### PR TITLE
build(env): added dev build tag

### PR DESCRIPTION
**Pull request by joaoReigota1**
Signed-off-by: João Reigota <joao.reigota@checkmarx.com>

**Proposed Changes**
- added `dev` build tag

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by rafaela-soares — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ The nightly release will have a "nightly" prefix with the last commit hash code.
 KICS is a true community project. It's built as an open source from day one, and anyone can find his own way to contribute to the project.
 [Check out how](docs/CONTRIBUTING.md), within just minutes, you can start making a difference, by sharing your expertise with a community of thousands of security experts and software developers.
 
+**Note:** For developing purposes not related to terraformer please use the Golang build tag `dev` in commands `go run, go build and go test`
```
Comment (nunoocx) [inline README.md:73]: "For development not related to terraformer..."
Maybe adding an example?

**Review by nunoocx — CHANGES_REQUESTED** [review]

**Review by nunoocx — DISMISSED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**